src/modules/common/components/__tests__/PageDetail.spec.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import React from 'react'
+import { shallow } from 'enzyme'
+import PageDetail from '../PageDetail'
+
+describe('PageDetail', () => {
+  const identifier = 'Date'
+  const information = 'May 22, 2020 1:00 AM'
+
+  it('should render', () => {
+    expect(shallow(<PageDetail identifier={identifier} information={information} />
+    )).toMatchSnapshot()
```

Comment:
I think we should just assert whether `'Date'` and `'May 22, 2020 1:00 AM'` exists in the DOM here.
Else this test will break because we switch the span to a div e.g.